src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -533,6 +545,13 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
           vertical-align: top;
           margin-left: var(--uom-margin-left);
         }
+        .more-info {
```

Comment:
Color it with the text color, which is not a full black. Maybe even secondary text color. We don't want it to attract too much attention